src/ontology/edits/oeo-physical.omn

```diff
@@ -309,6 +315,88 @@ Class: <http://openenergy-platform.org/ontology/oeo-physical/NuclearFuel>
         EnergyCarrier
     
     
+Class: <http://openenergy-platform.org/ontology/oeo-physical/PortionOfCarbonDioxide>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Carbon dioxide is a portion of matter with the chemical formula CO2. It is a gas that occurs naturally in the atmosphere as a trace gas and can work as a greenhouse gas.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "CO2"
+    
+    SubClassOf: 
+        PortionOfMatter,
+        <http://purl.obolibrary.org/obo/RO_0000091> some GreenhouseEffectDisposition
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo-physical/PortionOfFluorinatedGreenhouseGas>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A PortionOfFluorinatedGreenhouseGas is a PortionOfGreenhouseGas that is of anthropogenic origin."
```

Comment:
The current definition implies that all anthropogenic greenhouse gases are fluorinated but that is in reality not the case.
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A PortionOfFluorinatedGreenhouseGas is a PortionOfGreenhouseGas that is produced by fluorination. Hence they contains Fluor (F) atoms and are of anthropogenic origin."
```
I am aware that neither the terms _fluorination_ nor _fluor_ are part of the OEO and I don't think that it is necessary to include these. If we want to include these, we might link to ChEBI.